chain/types/ethtypes/eth_legacy_homestead_transactions.go

```diff
@@ -0,0 +1,369 @@
+package ethtypes
+
+import (
+	"fmt"
+
+	"golang.org/x/crypto/sha3"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	gocrypto "github.com/filecoin-project/go-crypto"
+	"github.com/filecoin-project/go-state-types/big"
+	typescrypto "github.com/filecoin-project/go-state-types/crypto"
+
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+const (
+	legacyHomesteadTxSignatureLen = 66
```

Comment:
We could also have one for eip1559 too to avoid the `65` magic number in that file

Reply:
Removed this and added exported constants for both the txn types.